### DISCUSSION TITLE
How can I not merge layers? I see the option 'tiff: write layers' in the code. How can I use it? The current command is:  magick 1.png --density 300 -background transparent -colorspace cmyk -flatten -compress lzw output2.tif

### DISCUSSION THREAD

**Discussion by 291598912** [ImageMagick/ImageMagick#6319] [Digital Image Processing]
How can I not merge layers? I see the option 'tiff: write layers' in the code. How can I use it? The current command is:

magick 1.png --density 300 -background transparent -colorspace cmyk -flatten -compress lzw output2.tif

**Comment by fmw42** [comment]
Remove the flatten. That is what merges layers.  Why do you use -flatten in your command? Perhaps you want

`magick 1.png --density 300 -background transparent -colorspace cmyk -alpha background -compress lzw output2.tif`

**Reply by 291598912** [reply]
magick 1.png  2.png 3.png 4.png 5.png --density 300 -background transparent -colorspace cmyk -flatten -compress lzw output2.tif

Merge multiple images into a single page TIF

**Reply by fmw42** [reply]
Did you read and try my command above?  Did that work to avoid the merge?
